README.md

```diff
@@ -781,6 +781,8 @@ In the [Setup](https://pkg.go.dev/github.com/vmware-labs/reconciler-runtime/reco
 
 The tracker will automatically expire a track request if not periodically renewed. By default, the TTL is 2x the resync internal. This ensures all tracked resources will naturally have the tracking relationship refreshed as part of the normal reconciliation resource. There is no need to manually untrack a resource.
 
+The tracked resource and its tracker are not temporally coupled. Tracking of a resource can start before it exists. Once it gets created or mutated, its tracker will be enqueued for reconciliation.
```

Comment:
**[thought]:** Maybe "temporally coupled" isn't plain enough or imprecise. Also, this section's initial paragraph mentions mutations, but I consider it harmless to repeat.